packages/api-server/api_server/routes/internal.py

```diff
@@ -15,6 +15,9 @@
 
 
 async def process_msg(msg: Dict[str, Any], fleet_repo: FleetRepository) -> None:
+    if "type" not in msg:
+        logger.debug("Ignoring message, 'type' must include in msg field")
```

Comment:
This should be `logger.warn`. While we are at it, let's also change line 23 to `logger.warn` as well. I think warn is more suitable than error because the source of the problem is not with api server, it is just warning that it received something unexpected.

Reply:
Done! https://github.com/open-rmf/rmf-web/pull/634/commits/9ce9b7470f547514071eb16a525a64b9ae682c47